src/map_field.cpp

```diff
@@ -1499,8 +1495,9 @@ void map::player_in_field( Character &you )
                     you.add_msg_player_or_npc( m_bad, _( "The acid burns your body!" ),
                                                _( "The acid burns <npcname>'s body!" ) );
                 } else if( total_damage > 0 ) {
-                    you.add_msg_player_or_npc( m_bad, _( "The acid burns your legs and feet!" ),
-                                               _( "The acid burns <npcname>'s legs and feet!" ) );
+                    you.add_msg_player_or_npc( m_bad,
+                                               _( "The acid burns your " ) + you.get_ground_contact_bodyparts_string() + "!",
+                                               _( "The acid burns <npcname>'s " ) + you.get_ground_contact_bodyparts_string() + "!" );
```

Comment:
You cannot translated construct strings through concatenation, they won't be translated correctly.
It needs to look like:
```cpp
string_format( _( "The acid burns your %s!" ), string_for_contact_body_part_types );
```